File: public/phase-2-20160620022203/C/equality.c
// Name : Equality
#include <stdio.h>
int main() {
	// input:
	// n :  the amount of array elements
	// m : array elements
	int n, m[10000], result=3, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &m[index]);
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%d", result);
	return 0;
}

File: public/phase-2-20160620022203/JAVA/equality.java
// Name : Equality
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n :  the amount of array elements
		// m : array elements
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] m = new int[n];
		String[] m_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			m[index] = Integer.parseInt(m_elements[index]);


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		int result=3;

		System.out.println(result);

	}
}